### PR TITLE
core(image-size-responsive): remove elidedUrl, elide url property instead

### LINKED ISSUE #13182: drop `image-size-responsive` elidedUrl
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/image-size-responsive.js#L149

This is the only audit that saves a separate "elidedUrl" property. All other audits save the result of `URL.elideDataURI` to their `url` property. We should just set `url` to this elided value, and drop `elidedUrl`. Otherwise, data uir images failing this audit will make the LHR larger than intended.

### PR DESCRIPTION
**Pull request by prerana1821**
**Summary**

The audit "image-size-responsive" used to contain "elidedUrl" property. All other audits save the result of URL.elideDataURI to their url property. So, I have set url to this elided value, and droped elidedUrl. I also have updated all the tests related to this audit.
image

**Resolved Issue**

fixes #13182

### CODE REVIEW

**Comment by prerana1821** [comment]
@adamraine I have done all the fixes mentioned.

**Comment by adamraine** [comment]
Thank you for making those fixes!

There are still some unrelated style changes in this PR (indentations, spacing, etc.). These changes just make the PR larger than it needs to be, especially in our generated files like `sample_v2.json`. Some of the style changes are good, but they still aren't really necessary.

TL;DR: If the changes aren't functional, we should leave them out of this PR :)

**Comment by prerana1821** [comment]
Hey @adamraine, I'm sorry for the unnecessary changes. I have completed the changes as per the review

**Comment by connorjclark** [comment]
Please take another look, there are still unnecessary whitespace changes.
